img/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>The Basic Language ogf the Web:HTML</title>
+        </head>
+        <body>
+            <style>
+                body {background: lightgrey;}
+                head {color:bisque;}
+                </style>
+                📘<p><h1><strong>The Code Magazine</p></h1></strong>
+
+<h2><p><b>The Basic Language of the Web: HTML</h2></p></b>
+<img src="laura-jones.jpg" alt="Laura-Jones" width="90" height="80">
+
+<h5><p>Posted by <b>Laura Jones</b> on Monday, June 21st 2027</h5></p>
+<img src="post-img.jpg" alt="screen post" width="650" height="230">
+
+<h5><p>All modern websites and web applications are built using three fundamental technologies: HTML, CSS and JavaScript. These are the languages of the web.</h5></p>
+
+<h5><p>In this post, let's focus on HTML. We will learn what HTML is all about, and why you too should learn it.</h5></p>
+
+<h3><p><b>What is HTML?</h3></p></b>
+
+<h5><p><b>HTML</b> stands for <b>H</b>yper<b>T</b>ext <b>M</b>arkup <b>L</b>anguage. It's a markup language that web developers use to structure and describe the content of a webpage (not a programming language). </h5></p>
```

Comment:
Your text is bold because everything is inside a `<h5>` tag. So the browser makes it bold and thinks that it is a title.

Just get rid of redundant `h` tags and you'll be fine